crates/bevy_render/src/primitives/mod.rs

```diff
@@ -81,17 +81,17 @@ impl Sphere {
     }
 }
 
-/// A plane defined by a unit normal and distance from the origin along the normal
-/// Any point `p` is in the plane if `n.p + d = 0`
-/// For planes defining half-spaces such as for frusta, if `n.p + d > 0` then `p` is on
+/// A half-space defined by a unit normal and distance from the origin along the normal
+/// Any point `p` is in the half-space if `n.p + d = 0`
+/// When defining half-spaces such as for frusta, if `n.p + d > 0` then `p` is on
```

Comment:
This needs to be rephrased. Specifically the following is wrong

> Any point `p` is in the half-space if `n.p + d = 0`

Specifically, `p` is in the half-space if `n.p + d > 0`.

Reply:
Adjusted the documentation, thank you